beekeeper-cleanup/src/main/java/com/expediagroup/beekeeper/cleanup/path/PathCleaner.java

```diff
@@ -15,7 +15,10 @@
  */
 package com.expediagroup.beekeeper.cleanup.path;
 
+import com.expediagroup.beekeeper.core.model.HousekeepingPath;
+
 public interface PathCleaner {
 
-  void cleanupPath(String housekeepingPath, String tableName);
+  void cleanupPath(HousekeepingPath housekeepingPath);
```

Comment:
We don't publish this as an API right, cause this is backward incompatible if just used internally we're good

Reply:
The only thing I can think of which would be affected is the vacuum tool, which I'll check now

Reply:
This is fine, the vacuum tool doesn't use the cleaner, just the scheduler which hasn't changed

Reply:
ok